dnf-docker-test/features/history_undo.feature

```diff
@@ -0,0 +1,101 @@
+Feature: DNF/Behave test history undo
```

Comment:
proper description is missing.. everyone knows that it is DNF test using behave and obviously everyone knows that this is a test